app/geo/gui/src/ts/twxplore/gui/geo/components/filterComponent/FilterComponent.tsx

```diff
@@ -86,34 +88,33 @@ const FilterComponentImpl: React.FunctionComponent<{featureType: string}> = ({
               }
               name={attributeName}
             />
-          </div>
+          </Grid>
         );
       }
       case TypeOfFeatureAttribute.STRING: {
         let stringAttributeState = attributeState as MapStringFeatureAttributeState;
         return (
-          <FormControl className={classes.formControl} key={attributeName}>
-            <Typography id="type" gutterBottom>
-              {attributeName}
-            </Typography>
-            <Autocomplete
-              multiple
-              id="tags-outlined"
-              options={stringAttributeState.values!}
-              getOptionLabel={option => option}
-              filterSelectedOptions
-              onChange={(event: any, value: string | string[]) => {
-                handleChangeSelect(event, value, filterIndexOfAttribute);
-              }}
-              renderInput={params => (
-                <TextField
-                  {...params}
-                  variant="outlined"
-                  placeholder="Select"
-                />
-              )}
-            />
-          </FormControl>
+          <Grid item xs={12} key={attributeName}>
```

Comment:
You only need key= when you generate a bunch of elements in a loop e.g., items.map(item => <Item key=...>), so React can distinguish them for ordering purposes.